presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1394,6 +1412,11 @@ public static boolean isJoinSpillingEnabled(Session session)
         return session.getSystemProperty(JOIN_SPILL_ENABLED, Boolean.class) && isSpillEnabled(session);
     }
 
+    public static boolean isAggregationSpillEnabled(Session session)
+    {
+        return session.getSystemProperty(AGGREGATION_SPILL_ENABLED, Boolean.class) && isSpillEnabled(session);
+    }
+
     public static boolean isDistinctAggregationSpillEnabled(Session session)
     {
         return session.getSystemProperty(DISTINCT_AGGREGATION_SPILL_ENABLED, Boolean.class) && isSpillEnabled(session);
```

Comment:
change this and orderby to check isAggregationSpillEnabled() instead of isSpillEnabled().  (or alternatively, just remove the isSpillEnabled check for both since we check all three properties anyway when build the aggregation operator)